content/en/data_streams/java.md

```diff
@@ -17,7 +17,7 @@ further_reading:
 
 | Technology     | Library                                                                                         | Minimal tracer version | Recommended tracer version |
 |----------------|-------------------------------------------------------------------------------------------------|------------------------|-----------------------------
-| Kafka          | [kafka-clients](https://mvnrepository.com/artifact/org.apache.kafka/kafka-clients)              | 1.9.0                  | 1.43.0 or later            |
+| Kafka          | [kafka-clients](https://mvnrepository.com/artifact/org.apache.kafka/kafka-clients) (v3.7 is not supported)              | 1.9.0                  | 1.43.0 or later            |
```

Comment:
to be exact, v3.7 is not fully supported (in many cases, it will still work)
Maybe we want to make it clear that everything after 3.8 is supported correctly?